src/lib/git.js

```diff
@@ -86,6 +86,11 @@ async function fetch(dir, branch) {
   );
 }
 
+async function fetchFromRemote(dir, name, url, branch) {
```

Comment:
since it seems to be always `upstream` this way there's no need to always add it and you can remove it from build-chain-flow-helper.js 
```suggestion
async function fetchFromRemote(dir, url, branch, name = "upstream") {
```

Reply:
Added!